File: 2.0/python/campaigns/getCampaignBatch.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.campaigns.getCampaignBatch(
    id=379506003,
    # return only specific fields
    fields='name,status,size'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/campaigns/getCampaignBatch.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getCampaignBatch();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "name,status,size"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
